src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { colorMap, size, fontSize } from './theme';
+import { P } from './text';
+
+const TAB_WIDTH = '136px';
+
+const Tab = styled('div')`
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    display: flex;
+    justify-content: space-between;
+`;
+
+const activeStyles = css`
+    border-bottom-color: ${colorMap.darkGreen};
+    color: ${colorMap.devWhite};
+`;
+
+const TabButton = styled('button')`
+    border: none;
+    background-color: inherit;
+    border-bottom: 3px solid transparent;
+    color: ${colorMap.greyDarkOne};
+    cursor: pointer;
+    font-size: ${fontSize.default};
+    font-family: 'Fira Mono';
+    &:hover {
+        color: ${colorMap.devWhite};
+    }
+    outline: none;
+    padding: ${size.micro} ${size.mediumLarge} ${size.medium};
```

Comment:
Just checked one last time and the bottom padding is only 16px! Would you mind updating that using `theme.js`?

Reply:
Not a problem!